src/chronometer.js

```diff
@@ -1,30 +1,38 @@
 class Chronometer {
-  constructor() {
-    // ... your code goes here
+  constructor()
+   {
+    this.currentTime = 0;
+    this.intervalId  = null;
   }
-
   start(printTimeCallback) {
-    // ... your code goes here
+      this.intervalId = setInterval(() => {
+      this.currentTime++
+      if (printTimeCallback) printTimeCallback ()},1000)
+    
   }
 
   getMinutes() {
-    // ... your code goes here
+   return Math.floor(this.currentTime/60)
   }
-
-  getSeconds() {
-    // ... your code goes here
+  getSeconds(){
+  if (this.currentTime === 0){
+    return 0;
+  } else {
+  return this.currentTime % 60
   }
+}
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    return value.toString().padStart(2, 0)
   }
 
-  stop() {
-    // ... your code goes here
+  stop(){
+    let clear= this.intervalId;
+    clearInterval(clear)
   }
 
   reset() {
-    // ... your code goes here
+   this.currentTime = 0; 
   }
 
   split() {
```

Comment:
Here for this `split()` method, you could use methods that you correctly built previously.
Something like this:
```
split() {
    let min = this.computeTwoDigitNumber(this.getMinutes());
    let sec = this.computeTwoDigitNumber(this.getSeconds());

    return `${min}:${sec}`;
  }
```